src/scales/scale.logarithmic.js

```diff
@@ -160,52 +147,30 @@ class LogarithmicScale extends Scale {
 		return this.getPixelForValue(ticks[index].value);
 	}
 
-	/**
-	 * Returns the value of the first tick.
-	 * @param {number} value - The minimum not zero value.
-	 * @return {number} The first tick value.
-	 * @private
-	 */
-	_getFirstTickValue(value) {
-		const exp = Math.floor(log10(value));
-		const significand = Math.floor(value / Math.pow(10, exp));
-
-		return significand * Math.pow(10, exp);
-	}
-
 	_configure() {
 		const me = this;
 		let start = me.min;
-		let offset = 0;
 
 		Scale.prototype._configure.call(me);
 
-		if (start === 0) {
-			start = me._getFirstTickValue(me.minNotZero);
-			offset = valueOrDefault(me.options.ticks.fontSize, defaults.fontSize) / me._length;
-		}
-
 		me._startValue = log10(start);
-		me._valueOffset = offset;
-		me._valueRange = (log10(me.max) - log10(start)) / (1 - offset);
+		me._valueRange = log10(me.max) - log10(start);
 	}
 
 	getPixelForValue(value) {
 		const me = this;
-		let decimal = 0;
-
-		if (value > me.min && value > 0) {
-			decimal = (log10(value) - me._startValue) / me._valueRange + me._valueOffset;
+		if (value === undefined || value === 0) {
+			value = me.min;
```

Comment:
can we return `null` here or something? this seems a little funny and I'm not quite sure why we're doing it

Reply:
Returning min value of scale for 0 (and parsed 0 that will be undefined), so those values are drawn.

Reply:
Why use the scale min instead of `undefined` though? It seems strange to me to change it to a somewhat arbitrary different value rather than making it `undefined`

Reply:
Lets make one thing clear, do you think we should continue to draw zero on a log chart or not?

Reply:
I would say we should not

Reply:
There are few fixed issues about that. #4913 fixed some for v2.7.1

If its not clear already, this is not something I want to spend my time with.

Reply:
Ok. I'm not sure how this will work in all cases, but if we need to change it later I suppose we can